hyrax/spec/helpers/nims/blacklight_helper_spec.rb

```diff
@@ -52,10 +52,16 @@
   describe '#render_truncated_description' do
     let(:publication) { FactoryBot.create(:publication) }
     let(:description) { ['A long description that is quite long. Long long long long long long long long long long. Extremely extremely extremely long long long long long long long.'] }
+    it 'has a "View More" button in catalog view' do
+      args = { value: description,
+               document: { controller: 'catalog', action: 'index' } }
+      p helper.render_truncated_description(args)
+      expect(Capybara.string(helper.render_truncated_description(args))).not_to have_link('View More')
+    end
     it 'has a "View More" button' do
```

Comment:
The revised test code causes discrepancy with the test description. This should be "doesn't have a View More button"